php/libraries/NDB_BVL_InstrumentStatus.class.inc

```diff
@@ -160,12 +160,12 @@ class NDB_BVL_InstrumentStatus
     /**
      * Sets the current data entry status of the instrument
      *
-     * @param string $status the new status from the set (NULL,
-     *                       'In Progress','Not Complete','Complete')
+     * @param ?string $status the new status from the set (NULL,
+     *                        'In Progress','Not Complete','Complete')
      *
      * @return void
      */
-    function setDataEntryStatus(string $status): void
+    function setDataEntryStatus(?string $status): void
```

Comment:
as stated in the description of the function, `$status` should be allowed to be `null`. These changes are necessary to accept null as a valid status.

Reply:
I don't think null should be a valid option.